gulpfile.cjs

```diff
@@ -583,6 +588,26 @@ function getMimeType(filename) {
     return { type: 'application/octet-stream', compress: true };
 }
 
+// get all files currently in bucket asynchronously
+function listAll(s3, bucketName, prefix, files, marker) {
```

Comment:
remove this function since it's not used anywhere

Reply:
Done